jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/panel/DynamicAttributesPanel.java

```diff
@@ -276,6 +283,22 @@ protected void onInstanceContainerItemChangeEvent(InstanceContainer.ItemChangeEv
         }
     }
 
+    @Override
+    public Optional<Component> findOwnComponent(String id) {
+        return getOwnComponents().stream()
+                .filter(component -> sameId(component, id))
+                .findAny();
+    }
+
+    @Override
+    public Collection<Component> getOwnComponents() {
+        Collection<Component> components = new ArrayList<>();
```

Comment:
1. Components returned by `propertiesForm.getOwnComponents()` are not own components of `DynamicAttributesPanel`. `propertiesForm` is.
2. You should return immutable collection, e.g. by `List.of(...)`